enterprise/signals.py

```diff
@@ -464,3 +464,14 @@ def generate_default_orchestration_record_display_name(sender, instance, **kwarg
 
 if COURSE_ENROLLMENT_CHANGED is not None:
     COURSE_ENROLLMENT_CHANGED.connect(course_enrollment_changed_receiver)
+
+
+@receiver(pre_save, sender=SAPSuccessFactorsEnterpriseCustomerConfiguration)
+def update_decrypted_credentials(sender, instance, **kwargs):     # pylint: disable=unused-argument
+    """
+    Ensure that the decrypted credentials have same values as unencrypted credentials.
+    """
+    if instance.key != instance.decrypted_key:
+        instance.encrypted_key = instance.key
+    if instance.secret != instance.decrypted_secret:
+        instance.encrypted_secret = instance.secret
```

Comment:
as per discussion in call, please update to refer model attributes here

Reply:
@sameenfatima78 updated.